app/src/main/java/org/oxycblt/auxio/util/SizeUtil.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2025 Auxio Project
+ * SizeUtil.kt is part of Auxio.
+ */
+
+package org.oxycblt.auxio.util
+
+import java.util.Locale
+
+/**
+ * Format a byte count into a human readable string (B, KB, MB, GB, ...).
+ */
+fun Long.formatFileSize(): String {
```

Comment:
There are utilities already available in android for measuring size so you don't necessarily need a vibecoded one @hanebane3904, can you fix this?